package.json

```diff
@@ -167,6 +168,7 @@
     "event-stream": "3.3.2",
     "expect.js": "0.3.1",
     "faker": "1.1.0",
+    "fs-extra": "0.30.0",
```

Comment:
Adds a few extra file system methods that aren't included in the native 'fs' module. It is used in the unit tests for copying directory to another path (also creates that path is not there), for emptying the contents of a directory and for removing directory.